Lib/test/test_xmlrpc.py

```diff
@@ -1171,6 +1171,26 @@ def test_gzip_decode_limit(self):
         xmlrpclib.gzip_decode(encoded, max_decode=-1)
 
 
+class HeadersServerTestCase(BaseServerTestCase):
+    class RequestHandler(xmlrpc.server.SimpleXMLRPCRequestHandler):
+        test_header = None
+
+        def do_POST(self):
+            self.__class__.test_header = self.headers.get('x-test')
+            return super().do_POST()
+    requestHandler = RequestHandler
```

Comment:
why this line?

Reply:
Because it is the attribute BaseServerTestCase is looking for in setUp to start the server.